### ISSUE TITLE
Remove the dinf and dref boxes from the table in Section 9.1.2

### ISSUE THREAD

**Issue by wantehchang** [AOMediaCodec/av1-avif#270]
The `dinf` and `dref` boxes declare the location of the media information in a track. I think they are unrelated to image items, so they should not appear in the table in Section 9.1.2 "Requirements on additional image item related boxes".

**Comment by leo-barnes** [comment]
I don't have any strong opinions here. I think MIAF restricts it so that external references are not allowed, so `dinf`/`dref` doesn't really serve any purpose in AVIF for image items. But on the other hand it is a relatively common box I guess so it might be good to document it.

@cconcolato, @podborski, @y-guyon , thoughts?

**Comment by y-guyon** [comment]
I would remove `dinf` and `dref` from 9.1.

Suggested new section 9.2 (copypasted from 9.1 with a few edits):

```bs
<h4 id="avif-required-boxes-additional-sequences">Requirements on additional image sequence related boxes</h4>

<p>The boxes indicated in the following table may be present in an [=AVIF file=] to provide signaling for image sequences. The boxes may be present inside the box indicated in the "Containing Box" column. <assert>If present, they shall use the version indicated in the table.</assert> [=/AVIF=] readers are expected to understand at least the boxes and versions listed in this non-exhaustive table. Additional boxes in the <code>'[=meta=]'</code> hierarchy may also be present.</p>
<table class="data">
    <thead>
        <tr>
            <th>Level 1</th>
            <th>Level 2</th>
            <th>Version(s)</th>
            <th>Specification</th>
            <th>Containing Box</th>
            <th>Description</th>
        </tr>
    </thead>
    <tbody>
        <tr>
            <td>[=dinf=]</td>
            <td>&nbsp;</td>
            <td>-</td>
            <td>[[!ISOBMFF]]</td>
            <td>[=meta=]</td>
            <td>Used to indicate the location of the media information in a track</td>
        </tr>
        <tr>
            <td>&nbsp;</td>
            <td>[=dref=]</td>
            <td>0</td>
            <td>[[!ISOBMFF]]</td>
            <td>&nbsp;</td>
            <td>&nbsp;</td>
        </tr>
    </tbody>
</table>
```

**Comment by leo-barnes** [comment]
@wantehchang @y-guyon 
Thinking on this some more, I don't think we should make these changes.

`dinf` is a sub-box **both** of `mdia` and `meta`. It can be used to point to external files, but **it can also be used to point to `IdentifiedMediaDataBox`es**. IIRC, MIAF only disallows pointing to external files. Using a `dinf` to point to an `IdentifiedMediaDataBox` would still be perfectly valid in AVIF (although I doubt it's implemented anywhere).

I have changed the wording in #273 to indicate that it's not only for tracks.

**Comment by wantehchang** [comment]
Leo: Thanks for the reply. In ISOBMFF the descriptions of 'dinf' and 'dref' contain the word "track", but in HEIF Clause 10.2.2.2 Requirements on readers, Table 3, the description of 'dref' is "data reference box, declares source(s) of items".

So we just need to change the description of 'dinf' in our table to replace "track" with "item"?

**Comment by wantehchang** [comment]
Leo: Actually, in ISOBMFF, the description of 'dref' in Clause 6.3.4 Box order, Table 1, when under the 'meta' box, is "data reference box, declares source(s) of metadata items".

So is 'dref' only used with metadata items and not used with image items?

**Comment by leo-barnes** [comment]
> So we just need to change the description of 'dinf' in our table to replace "track" with "item"?

I think so, yes.

> So is 'dref' only used with metadata items and not used with image items?

The `meta` box was initially (pre-HEIF) only meant to store metadata items if I remember my history correctly. With HEIF, image items got added. I think this should simply just say "items" now. I'll file an issue in the ISOBMFF tracker.

**Comment by leo-barnes** [comment]
I'll mark this as getting fixed by #273, which changes the text for `dinf` to "Used to indicate the location of the media information".

Please reopen if you disagree. :)
